v21.0 does not install all the required headers

**What version of protobuf and what language are you using?**

Version: v21.0
Language: C++

**What operating system (Linux, Windows, ...) and version?**

Linux, but I assume it affects all operating systems

**What runtime / compiler are you using (e.g., python version or gcc version)**

```
 gcc --version
gcc (Ubuntu 11.2.0-19ubuntu1) 11.2.0
Copyright (C) 2021 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

**What did you do?**

Compiled and installed Protobuf using CMake.  To workaround #10045 I used the CMake [`FindProtobuf` module](https://cmake.org/cmake/help/latest/module/FindProtobuf.html).

**What did you expect to see**

I would expect a simple program that includes `google/protobuf/message.h` to compile.

**What did you see instead?**

The program does not compile with this error:

```
18 0.913 /usr/bin/c++    -MD -MT CMakeFiles/main.dir/test.cc.o -MF CMakeFiles/main.dir/test.cc.o.d -o CMakeFiles/main.dir/test.cc.o -c /var/tmp/build/test/test.cc
#18 0.913 In file included from /usr/local/include/google/protobuf/unknown_field_set.h:53,
#18 0.913                  from /usr/local/include/google/protobuf/generated_message_reflection.h:47,
#18 0.913                  from /usr/local/include/google/protobuf/message.h:124,
#18 0.913                  from /var/tmp/build/test/test.cc:1:
#18 0.913 /usr/local/include/google/protobuf/parse_context.h:45:10: fatal error: google/protobuf/endian.h: No such file or directory
#18 0.913    45 | #include <google/protobuf/endian.h>
#18 0.913       |          ^~~~~~~~~~~~~~~~~~~~~~~~~~
#18 0.913 compilation terminated.
```

**Anything else we should know about your project / environment**

You can use [Dockerfile-2.txt](https://github.com/protocolbuffers/protobuf/files/8775278/Dockerfile-2.txt) to repro the problem, just run:

```shell
docker buildx build --progress plain -t test - <Dockerfile-2.txt
```

--- a/cmake/extract_includes.bat.in
+++ b/cmake/extract_includes.bat.in
@@ -51,6 +51,7 @@ copy "${PROTOBUF_SOURCE_WIN32_PATH}\..\src\google\protobuf\descriptor_database.h
 copy "${PROTOBUF_SOURCE_WIN32_PATH}\..\src\google\protobuf\duration.pb.h" include\google\protobuf\duration.pb.h
 copy "${PROTOBUF_SOURCE_WIN32_PATH}\..\src\google\protobuf\dynamic_message.h" include\google\protobuf\dynamic_message.h
 copy "${PROTOBUF_SOURCE_WIN32_PATH}\..\src\google\protobuf\empty.pb.h" include\google\protobuf\empty.pb.h
+copy "${PROTOBUF_SOURCE_WIN32_PATH}\..\src\google\protobuf\endian.h" include\google\protobuf\endian.h
 copy "${PROTOBUF_SOURCE_WIN32_PATH}\..\src\google\protobuf\explicitly_constructed.h" include\google\protobuf\explicitly_constructed.h
 copy "${PROTOBUF_SOURCE_WIN32_PATH}\..\src\google\protobuf\extension_set.h" include\google\protobuf\extension_set.h
 copy "${PROTOBUF_SOURCE_WIN32_PATH}\..\src\google\protobuf\extension_set_inl.h" include\google\protobuf\extension_set_inl.h

